### ISSUE TITLE
unsolved reference: Web3jFactory

### ISSUE THREAD

**Issue by aaaking** [web3j/web3j#651]
I add api ('org.web3j:core:3.3.1-android') in build.gradle , but still error:
unsolved reference: Web3jFactory

**Comment by iikirilov** [comment]
@aaaking is this still an issue? Did you refresh your dependencies?

**Comment by dimitrileite** [comment]
As of the fact that the Lib was updated. I needed to recreate the Class: Web3jFactory under some Util directory in my App`s folder. I worked.

**Comment by iikirilov** [comment]
https://docs.web3j.io/getting_started.html#start-sending-requests - the last example in this section should help. Please reopen if still an issue.

**Comment by JoaquimLey** [comment]
@iikirilov It seems like the Web3jFactory is missing from `v4.1.0-android` 🤔, can you confirm this? 
Btw this issue seems to be only present for someone using `Kotlin`, in my use case I have a plain old Java library and still complains due to `prohibited static interface`.

----------------

For anyone reading this issue and using Kotlin, feel free to create your own `Web3jFactory` **Java class**:

```java

import org.web3j.protocol.Web3j;
import org.web3j.protocol.Web3jService;
import org.web3j.protocol.http.HttpService;

import java.util.concurrent.ScheduledExecutorService;

public class Web3jFactory {

    public static Web3j build(String endpoint) {
        return Web3j.build(new HttpService(endpoint));
    }


    public static Web3j build(Web3jService web3jService, long pollingInterval,
                       ScheduledExecutorService scheduledExecutorService) {
        return Web3j.build(web3jService, pollingInterval, scheduledExecutorService);
    }
}
```

**Comment by iikirilov** [comment]
@snazha-blkio any insight on this? How do build an instance with web3j-android? What is the latest version?

**Comment by snazha-blkio** [comment]
> @snazha-blkio any insight on this? How do build an instance with web3j-android? What is the latest version?

Looks like Web3jFactory for android last appears in [2.1.0-android](https://github.com/web3j/web3j/blob/v2.1.0-android/src/main/java/org/web3j/protocol/Web3jFactory.java), but disappeared in [3.0.0-android](https://github.com/web3j/web3j/blob/v3.0.0-android/src/main/java/org/web3j/protocol/Web3jFactory.java). 

This highlights an issue in the documentation, which I will get corrected.

Users now ought to be using the [Web3j](https://github.com/web3j/web3j/blob/v4.1.0-android/core/src/main/java/org/web3j/protocol/Web3j.java) interface, which brings the android version inline with the mainstream version.

The latest official release of web3j is 4.1.0-android, and the latest snapshot is 4.1.2-android-SNAPSHOT.

**Comment by snazha-blkio** [comment]
I have raised a PR to address the incorrect documentation.
https://github.com/web3j/web3j/pull/887

**Comment by snazha-blkio** [comment]
PR#887 has been merged, please use version 4.2.0+ hereon.
